IaC/AKS/cluster/charts/ngsa/templates/deployment.yaml

```diff
@@ -47,25 +49,25 @@ spec:
             httpGet:
               path: /healthz
               port: http
-            initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds }}  
-            failureThreshold: {{ .Values.livenessProbe.failureThreshold }}  
-            periodSeconds: {{ .Values.livenessProbe.periodSeconds }}  
+            initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds }}
```

Comment:
do we need to define a liveness probe for webv?  since it's not an http server, not sure what that would be

Reply:
Yup, since it's not a server it wouldn't make sense.

Reply:
We found the `exec` option that looks promising for non http services. We'll make a new triage item to investigate.
https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/

Reply:
References #106